Makefile

```diff
@@ -76,11 +76,30 @@ integrationtest-postgres: build-postgresql
 integrationtest-rabbitmq: build-rabbitmq build-sda
 	@PR_NUMBER=$$(date +%F) docker compose -f .github/integration/rabbitmq-federation.yml run federation_test
 	@PR_NUMBER=$$(date +%F) docker compose -f .github/integration/rabbitmq-federation.yml down -v --remove-orphans
-integrationtest-sda: build-all
```

Comment:
could it make sense to keep the target `integrationtest-sda` as an way to run all integration tests? s3, posix and sync.